knowing-more-about-ourselves.rst

```diff
@@ -8,58 +8,63 @@
       Collect and analyze project and build data to make better decision about our product
       with a strong focus on the user experience.
 
-Knowing more about ourselves
-============================
+Knowing more about how people use our service
+=============================================
 
-Read the Docs generates a lot of data on it owns.
+Read the Docs generates a lot of data.
 We are the largest documentation platform out there,
-with hundred thousands projects using our product every day to host their documentation.
+with hundreds of thousands of projects using our product every day to host their documentation.
```

Comment:
This still reads weirdly, but I couldn't think of an obvious way to state it better. Maybe:

`with 250,000+ projects`

Reply:
I don't have a preference here. Both of them are readable to my level of English 😄